libs/ledger-live-common/src/families/cosmos/js-synchronisation.ts

```diff
@@ -54,130 +54,110 @@ const txToOps = (info: AccountShapeInfo, accountId: string, txs: CosmosTx[]): Op
 
     const correspondingMessages = messages.filter(m => m.type === mainMessage.type);
 
-    if (correspondingMessages.length === 0) {
```

Comment:
this check is not needed. I think. It never happens.

Reply:
I trust your analysis, let's remove it ! If i remember correctly it was more as a safety net, same as the loop in reward section